e2e/test/scenarios/onboarding/search/search.cy.spec.js

```diff
@@ -454,10 +697,13 @@ function getSearchBar() {
   return cy.findByPlaceholderText("Search…");
 }
 
-function generateQuestions(prefix, count) {
-  return _.range(count).map(idx => ({
-    name: `${prefix} ${idx + 1}`,
-    query: { "source-table": ORDERS_ID },
-    collection_id: null,
-  }));
+function expectSearchResultItemNameContent({ itemNames }) {
```

Comment:
Added this utility function to help me out with checking searches with `created_by` and `last_edited_by` - essentially, it checks that the search results only have titles that match that of the `itemNames`. 

In these tests, I've decided to use the `Reviews` table as the "non-filtered" element, since it's created by the database so it won't pop up as `created_by` or `last_edited_by` a user, and it's the only one with `Reviews`. So, in essence, there will only be 3 search results max: the reviews table, the admin's question (Admin's Super Duper Reviews), and the normal user's question (Robert's Super Duper Reviews).

If you think there's a problem with this approach please let me know, I found it to be the simplest way to test if things were working.